app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,34 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.all
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    new_food = Food.create(name: food_params[:name], measurement_unit: food_params[:measurement_unit],
+                           price: food_params[:price], user_id: current_user.id)
+    respond_to do |format|
+      format.html do
+        if new_food.save
+          flash[:notice] = 'Food successfully created'
+          redirect_to '/foods'
+        else
+          flash[:notice] = 'It was not possible to save the Food'
+          redirect_to '/foods/new'
+        end
+      end
+    end
+  end

```

Comment:
- [ ] The food is currently not saving in the database and it's rollbacking. Please check and fix that.

![Screenshot from 2021-11-27 20-49-07](https://user-images.githubusercontent.com/22434660/143687138-e6ba0c29-f91f-4760-b5db-da7de4c32961.png)

Reply:
the error is now fixed, for the next time you want to create a food the available measurement_unit are the ones in the foods modal 

inclusion: { in: ['Teaspoon (tsp)', 'Cup (c)', 'Unit (u)', 'Milliliter (ml)', 'Litter (l)',
                              'Milligram (mg)', 'Kilogram (kg)'] }

Reply:
![image](https://user-images.githubusercontent.com/44306286/143689015-d7c8dfee-4f2a-4da4-8cfe-1147f3e18db6.png)